pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -39,38 +39,36 @@ type topologyPairToScore map[string]map[string]int64
 
 type podAffinityPriorityMap struct {
 	topologyScore         topologyPairToScore
-	pod                   *v1.Pod
-	affinityTerms         []*processedWeightedAffinityTerm
-	antiAffinityTerms     []*processedWeightedAffinityTerm
-	sharedLister          schedulerlisters.SharedLister
+	affinityTerms         []*weightedAffinityTerm
+	antiAffinityTerms     []*weightedAffinityTerm
 	hardPodAffinityWeight int32
 	sync.Mutex
 }
 
-type processedWeightedAffinityTerm struct {
+type weightedAffinityTerm struct {
```

Comment:
worth adding a comment indicating that this the representation of v1.WeightedPodAffinityTerm

Reply:
done.